### PR TITLE
Adding R and RStudio icons

### PR DESCRIPTION
**Pull request by csgillespie**

### CODE REVIEW

**Comment by ferminrp** [comment]
I'm getting an error trying to import this. Can you check?

**Comment by csgillespie** [comment]
@ferminrp I didn't export the lib file. Fixed now

File: libraries.json [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+    "description": "R and RStudio icons.",
+    "authors": [
+      {
+        "name": "Jumping Rivers",
+        "url": "https://www.jumpingrivers.com"
+      }
+    ],
+    "source": "jumpingrivers/r.excalidrawlib",
+    "preview": "jumpingrivers/r.png",
+    "date": "2021-08-22"
```
Comment (dwelle) [inline libraries.json:13]:
```suggestion
    "date": "2021-08-23"
```

Comment (dwelle) [reply libraries.json:13]: mhm.. can't commit. Let's merge with `22`

**Comment by dwelle** [comment]
Thanks!

**Pull request merged**